IdentityCore/src/broker_operation/response/browser_native_message_response/MSIDBrowserNativeMessageGetTokenResponse.m

```diff
@@ -77,6 +77,11 @@ - (NSDictionary *)jsonDictionary
     response[@"account"] = accountJson;
     response[@"state"] = self.state;
     
+    __auto_type propertiesJson = [NSMutableDictionary new];
+    // TODO: once ests follow the latest protocol, this should be removed. Account ID should be read from accountJson.
```

Comment:
nit: Please consider to add a PBI

Reply:
https://identitydivision.visualstudio.com/Engineering/_boards/board/t/Auth%20Client%20-%20Apple/Backlog%20items/?workitem=2938432